meta-facebook/minerva-ag/src/platform/plat_isr.c

```diff
@@ -157,46 +157,29 @@ void check_clk_handler()
 	if (pwrgd_p3v3_and_p1v8 == 3) {
 		memset(data, 0, sizeof(data));
 		memcpy(data, (uint8_t[]){ 0x00, 0x01, 0x00, 0x00 }, 4);
-		if (!cpld_i2c_write(I2C_BUS1, AEGIS_312M_CLK_GEN_ADDR, 0xFC, data, 4)) {
+		if (!plat_i2c_write(I2C_BUS1, AEGIS_312M_CLK_GEN_ADDR, 0xFC, data, 4)) {
 			LOG_ERR("Failed to write 312M CLK GEN");
 			return;
 		}
-		if (!cpld_i2c_read(I2C_BUS1, AEGIS_312M_CLK_GEN_ADDR, 0x04, data, 1)) {
```

Comment:
@LisaChang-Quanta I am merging this PR, but for future can you please add relevant details into description mentioning what you are fixing and how it will resolve the issue u r trying to fix? Some relevant runtime UT will also be good.